packages/pn-landing-webapp/.env

```diff
@@ -5,4 +5,6 @@ NEXT_PUBLIC_PAGOPA_HOME = https://www.pagopa.it
 NEXT_PUBLIC_PRIVACY_DOCUMENT_PATH = /static/documents/informativa-privacy.pdf
 NEXT_PUBLIC_PA_PRIVACY_DOCUMENT_PATH = /static/documents/informativa-privacy-enti.pdf
 NEXT_PUBLIC_PA_PARTNERS_PATH = /static/documents/partners-and-intermediaries-list.pdf
-NEXT_PUBLIC_ONE_TRUST_PORTAL_CDN = https://privacyportalde-cdn.onetrust.com/77f17844-04c3-4969-a11d-462ee77acbe1/privacy-notices/b5c8e1dc-89df-4ec2-a02d-1c0f55fac052.json
\ No newline at end of file
+NEXT_PUBLIC_ONE_TRUST_BASEPATH = https://privacyportalde-cdn.onetrust.com/77f17844-04c3-4969-a11d-462ee77acbe1/privacy-notices
+NEXT_PUBLIC_ONE_TRUST_DRAFT_MODE = ''
```

Comment:
I should convert this variable to boolean. And only if it is defined and set to true, we append 'draft' to one trust cdn url (see DISABLE_INACTIVITY_HANDLER)